go.mod

```diff
@@ -34,6 +34,7 @@ require (
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/docker/docker v24.0.6+incompatible
 	github.com/docker/libnetwork v0.8.0-dev.2.0.20210525090646-64b7a4574d14
+	github.com/dustin/go-humanize v1.0.1
```

Comment:
Is it worth adding a new dependency merely for two log statements in a test?

Reply:
Changed it with a handmade conversion function :+1: